ui/css/perf.css

```diff
@@ -242,18 +242,35 @@ to { opacity: 0; }
   width: 10px;
 }
 
-.compare-improvement {
+.compare-table td {
+    text-align: center;
+    cursor: default;
+}
+
+.compare-table td.test-title {
+    text-align: left;
+}
+
```

Comment:
I would also add the following css, to make things a little nicer looking/easier to read…

Change line 112 to:
```css
.subtest-header {
  background-image: linear-gradient(to bottom, #EAEFF2, #D4DDE4);
  color: #CCC;
  font-weight: bold;
}
```

Near line 245, add:
```css
.compare-table .subtest-header > td {
  color: transparent;
  transition: color 150ms ease-in-out;
}

.compare-table:first-child .subtest-header > td,
.compare-table .subtest-header > td.test-title {
  color: #333;
}

.compare-table .subtest-header > td.test-title {
  text-align: left;
}
```

(If you don't want to always show the first legend, the latter two rules could be simplified to:
```css
.compare-table .subtest-header > td.test-title {
  color: #333;
  text-align: left;
}
```
)